artifacts-wg/charter/charter.md

```diff
@@ -0,0 +1,44 @@
+# Purpose
+In the distributed world of cloud-native computing different artifacts and packages are used to transport configuration and code for the many services and capabilities that comprise and support workloads and applications. As an example, today ArtifactHub advertises that it can crawl about 20 such artifact types [1]. These "cloud-native" bundles add to the previous proliferation of package and artifact types for software dependencies such as Maven and npm and for system packages like deb and rpm.
+
+This abundance of package types and formats adds complexity and risk for cloud application developers and architects trying to provision and run cloud-native workloads. Different tools and controllers must be carefully learned and installed to bundle, unbundle, verify and deploy each artifact type. This complexity slows product development by users, impedes innovation and collaboration amongst projects, and increases risks of insecure configuration.
+
+To reduce this complexity and facilitate collaboration and innovation, WG Artifacts will gather stakeholders from many CNCF and open source projects offering packaging, distribution and deployment mechanisms for bundles of configuration and code.
+
+**The WG will:**  
+* **Develop common patterns**  
+*  **Advocate for innovative projects**  
+*  **Gather end user feedback**
+**for packaging, discovery, distribution, and deployment mechanisms for** **cloud-native software artifacts.**
+
+# Alignment with TAG App Delivery and CNCF
+TAG App Delivery's charter [2] calls on the TAG to develop best practices, foster collaboration between projects, improve interoperability and propose new initiatives and projects related to the following domains which this working group would target.
+
+* Application bundling and deployment
+* Package management
+* Application delivery workflow and strategy
+* Configuration source-driven workflow
+* Release management
+
+The work on extensions, patterns and prototypes in this CNCF TAG complements work on specifications in the Open Containers Initiative (OCI) itself.
```

Comment:
Recommend removing the explicit mention of OCI as it is a bit too implementation specific. While yes, OCI is certainly an initial target, should it be in the charter for the WG?

Reply:
I wouldn't mind keeping it as `this CNCF TAG complements work in other projects like OCI`

Reply:
Agree with @sabre1041 and @sajayantony  . I think this WG should not be limited to specifications in OCI only, it would be flexible to mention that OCI spec is an example initiative. I guess the Artifact WG may also has its own Spec or contribute to other project specifications in the future.

Reply:
@sabre1041 @FeynmanZhou makes sense, I agree this line makes it sound like we're focusing exclusively on OCI bundles.

Perhaps we could add to the end of the sentence: 

"The work ... in this CNCF TAG complements work on specifications in ~~the~~ Open Containers Initiative (OCI) ~~itself~~, but will consider other packaging formats and not be limited only to OCI."